salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
```

Comment:
For the moment we don't have "metalk8s" section in the pillar and i don't know if we want it as we use pillarenv.
For me looks strange to have pillar like that as some information from a specific step may be needed for another.

Reply:
I personnaly prefer to structure the pillar as component section
```
kubelet:
  <kubelet stuff>
kube-apiserver:
  <kube-apiserver stuff>
preflight:
  <preflight stuff>
```
Basically preflight is more about all section but have a preflight section seems good to me also.

But it's only my opinion and it's can/should be discussed :)

Reply:
No I think you're right.
Maybe prefix everuthing related to kubeadm by ```kubeadm``` to isolate them, but since you're more "into" it, I trust you on this one.
We can always change later on the data until we release anyway. And since we will generate dynamically the pillars, changing the structure should'nt be that problematic anyway.